src/ops/log.rs

```diff
@@ -4,7 +4,10 @@ use derive_more::Display;
 use git2::Oid;
 use std::{ffi::OsString, rc::Rc};
 
-pub(crate) const ARGS: &[Arg] = &[];
+pub(crate) const ARGS: &[Arg] = &[
+    Arg::new_int_opt("-n", "Limit number of commits", Some(256)),
+    // Arg::new_str("-S", "Search occurences"), // TOOD: Implement search
```

Comment:
Just a reminder. `-S` is added in `src/default_config.toml`, not sure if it'd be visible in the UI. But lets either include the keybinding & impl, or none.